content/projects/data-science-specific/natural-language-processing/_index.md

```diff
@@ -24,26 +24,41 @@ tags:
 - sentiment-analysis
 - nltk
 - textblob
-title: Natural language processing
+title: Natural language processing.
 ---
+In order to help you develop your NLP skills, we have prepared two datasets for you to work with.These datasets are designed to improve your understanding of text processing, sentiment analysis, and text classification. You have the option to work on either one dataset or both datasets. Please follow the instructions below to get started:
 
-The contents of the State of the Nation Address (SONA) for every year dating back to 1990 is available on the [South African Government website](https://www.gov.za/state-nation-address). This gives us a great opportunity to look at the priorities and challenges have faced over time, and the focus points for the various presidents over this time.
+## Dataset 1: [Movie Reviews Sentiment Analysis](https://www.kaggle.com/datasets/lakshmi25npathi/imdb-dataset-of-50k-movie-reviews)
 
-1. Create a corpus from the English-language text for the SONAs dating back to 2000. Save them with the speaker information and date for later analysis. Where there is more than one SONA per year, get both.
+### Description: 
+This dataset contains a collection of movie reviews, each labeled with its corresponding sentiment (positive or negative). This is a dataset for binary sentiment classification containing substantially more data than previous benchmark datasets. We provide a set of 25,000 highly polar movie reviews for training and 25,000 for testing. So, predict the number of positive and negative reviews using Logistic Regression or other classification algorithms.
+For more dataset information, please go through the following [link](http://ai.stanford.edu/~amaas/data/sentiment/)
```

Comment:
I think the wording of this could be changed slightly

Instead of this:

We provide a set of 25,000 highly polar movie reviews for training and 25,000 for testing. So, predict the number of positive and negative reviews using Logistic Regression or other classification algorithms

This:

The dataset contains 25,000 highly polarised movie reviews for training, and 25,000 for testing. Predict the number of positive and negative reviews using Logistic Regression or other classification algorithms